### ISSUE TITLE
Support RFC 8738

### ISSUE THREAD

**Issue by bugproof** [fszlin/certes#294]
https://datatracker.ietf.org/doc/html/rfc8738

**Comment by webprofusion-chrisc** [comment]
Are you using an internal ACME CA, if so which one? The public CAs don't support IP certificates via ACME.

**Comment by bugproof** [comment]
> Are you using an internal ACME CA, if so which one? The public CAs don't support IP certificates via ACME.

Yes I'm using the public CA. I just realized that none of them support IPs. Only ZeroSsl's REST API which is limited to 3 certs per account. I was just about to make a PR for this. Open this again if you wish to ! Thanks.
